service/src/main/java/bio/terra/workspace/service/workspace/flight/CreateWorkspaceStep.java

```diff
@@ -39,6 +40,10 @@ public StepResult doStep(FlightContext flightContext)
       if (!workspace.equals(existingWorkspace)) {
         throw ex;
       }
+    } catch (DuplicateUserFacingIdException ex) {
```

Comment:
why catch this just to throw it?

Reply:
Done. (This was leftover from when I used the same exception for both cases.)